website/content/docs/concepts/workers.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+page_title: Workers
+description: |-
+  Introduction to Boundary workers
+---
+
+# Workers
+
+## Overview
+Boundary's architecture consists of 2 main components:
+1. **Control Plane** - made up of controllers
+2. **Data Plane** - made up of workers
+3. **Client** - installed on the user's device
+
+**Controllers** are what users authenticate to through the client, and contain Boundary's resources and permissions. In addition, controllers also communicate with external components 
+such as the database, KMS, Vault, identity providers, and plugins. **Workers** are primarily used as network proxies for Boundary sessions, allowing users to access 
+private targets. Instead of exposing a private network to the public, or allowing users to have access to entire private networks, workers create a direct network 
+tunnel between users and targets. 
+
+![IAM Example Structure](/img/access-model.png)
+
+## Use cases
```

Comment:
I think we need a sentence between the heading and the use cases to act as a transition. What about something like?

> You can configure workers in various ways depending on your needs. Consider the following use cases.

Reply:
Good idea